config/deploy.rb

```diff
@@ -6,10 +6,10 @@
 
 set :home_directory, "/home/lyberadmin"
 
-set :branch, 'master'
+#set :branch, 'master'
```

Comment:
Shouldn't this leave the branch set to master as the default instead of requiring the person doing the deploy to provide a branch name?